app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -181,35 +179,35 @@ class SuggestedEditsTasksFragment : Fragment() {
 
             displayedTasks.add(addDescriptionsTask)
             addDescriptionsTask.unlockMessageText = String.format(getString(R.string.suggested_edits_task_translate_description_edit_disable_text), targetForAddDescriptions)
-            addDescriptionsTask.showActionLayout = !Prefs.isSuggestedEditsAddDescriptionsUnlocked()
-            addDescriptionsTask.disabled = !Prefs.isSuggestedEditsAddDescriptionsUnlocked()
+            addDescriptionsTask.showActionLayout = false
+            addDescriptionsTask.disabled = false
```

Comment:
Same here, you can remove them since both default values are `false`.
```
in SuggestedEditsTask
    var disabled: Boolean = false
    var showActionLayout: Boolean = false
```